HOMEWORK_9/EX4.CPP

```diff
@@ -0,0 +1,37 @@
+#include <iostream>
+#include <cstring>
+
+void parseStringLetters(const char str[], int& vowelsCount, int& constonantsCount)
+{
+    int lenth = std::strlen(str);
+    for(int i = 0; i < lenth; i++)
+    {
+        if(str[i] >= 'a' && str[i] <= 'z')
+        {
+            constonantsCount++;
+        }
+        else if(str[i] >= 'A' && str[i] <= 'Z')
+        {
+            vowelsCount++;       
+        }
```

Comment:
consonants and vowels, not lowers and Uppers 🥲

Reply:
I think it looks better